packages/portal/docs/style/Typography.md

```diff
@@ -0,0 +1,80 @@
+More info about font styles and examples where they are applied can be found on the [Typography page on Figma](https://www.figma.com/file/TQHeWavFSkWCWGMntlthcJ/Component-Library?type=design&node-id=2330%3A6894&mode=design&t=7ZfmXiyHHKs1NQqO-1) and the [Font styles page in Confluence](https://europeana.atlassian.net/wiki/spaces/EC/pages/2279047169/Font+styles)
```

Comment:
Do you think it's worth moving the content of the font styles pages from Confluence to here? That way we don't have this information spread about in so many different places, especially as neither the Figma nor Confluence pages are publicly accessible.

Reply:
Good idea. I have updated to the typography table to include the info from the Confluence page. Each style now has a details row (togglable display) with more info on responsive styles and example use cases.